assets/js/blocks/mini-cart/index.tsx

```diff
@@ -33,6 +33,10 @@ const settings: BlockConfiguration = {
 		html: false,
 		multiple: false,
 		color: true,
+		typography: {
+			fontSize: true,
+			__experimentalFontFamily: true,
```

Comment:
The experimental feature should be enabled only for the featuring plugin. We can do it, in this way:
```suggestion
				...( isFeaturePluginBuild() && {
				__experimentalFontFamily: true,
			} ),
```

Reply:
Hmm, this actually broke the block. It would no longer show on the list of blocks 🤔

Reply:
Mmm, strange. Did you import the function?

Reply:
Yup. Forgot about the import! :)